src/Files.Core.SourceGenerator/Files.Core.SourceGenerator.csproj

```diff
@@ -19,8 +19,10 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="MarcosLopezC.LightJson" Version="0.5.3" PrivateAssets="all" />
```

Comment:
We're trying hard to avoid additional references especially from third parties. Fyi @0x5bfa

Reply:
Source generator affects package size?